release notes/v0.26.0.md

```diff
@@ -0,0 +1,132 @@
+
+### JSON export of the end-of-test summary report (#1168)
+
+This returns (from the very early days of k6) the ability to output the data from the end of test summary in a machine-readable JSON file.
+This report can be enabled by the `--summary-export <file_path>` CLI flag or the `K6_SUMMARY_EXPORT` environment variable. The resulting JSON file will include data for all test metrics, checks and thresholds.
+
+### New CSV output (#1067)
+
+There is an entirely new `csv` output that can be enabled by using the `--out csv` CLI flag. There are two things that can be configured: the output file with `K6_CSV_FILENAME` (by default it's `file.csv`), and the interval of pushing metrics to disk, which is configured with `K6_CSV_SAVE_INTERVAL` (1 second by default). Both of those can be configured by the CLI as well: `--out csv=somefile.csv` will output to `somefile.csv` and `--out file_name=somefile.csv,save_interval=2s` will output again to `somefile.csv`, but will flush the data every 2 seconds instead of every second.
+
+The first line of the output is the names of columns and looks like:
+```
+metric_name,timestamp,metric_value,check,error,error_code,group,method,name,proto,status,subproto,tls_version,url,extra_tags
+http_reqs,1573131887,1.000000,,,,,GET,http://httpbin.org/,HTTP/1.1,200,,,http://httpbin.org/,
+http_req_duration,1573131887,116.774321,,,,,GET,http://httpbin.org/,HTTP/1.1,200,,,http://httpbin.org/,
+http_req_blocked,1573131887,148.691247,,,,,GET,http://httpbin.org/,HTTP/1.1,200,,,http://httpbin.org/,
+http_req_connecting,1573131887,112.593448,,,,,GET,http://httpbin.org/,HTTP/1.1,200,,,http://httpbin.org/,
+```
+
+All thanks to @Sirozha1337!
+
+
+### JSON output optimizations (#1114)
+
+The JSON output no longer blocks the goroutine sending samples to the file, but instead (like all other outputs) buffers the samples and writes them at regular intervals (100ms and is currently not configurable). It also uses a slightly faster way of encoding the data, which should decrease the memory usage by a small amount.
+
+Another improvement is the ability to compress the generated JSON file by simply adding `.gz` to the end of the file name. Compressed files are typically 30x smaller.
+
+### InfluxDB output improvements (#1113)
+
+The InfluxDB output has been updated to use less memory and try to send smaller and consistent chunks of data to InfluxDB, in order to not drop packages and be more efficient. This is primarily done by sending data in parallel, as this seems to be better from a performance perspective, and more importantly, queuing data in separate packets, so that we don't send the data for a big time period all at once. Also, the used library was updated, which also decreased the memory usage.
```

Comment:
```suggestion
The InfluxDB output has been updated to use less memory and try to send smaller and consistent chunks of data to InfluxDB, in order to not drop packets and be more efficient. This is primarily done by sending data in parallel, as this seems to be better from a performance perspective, and more importantly, queuing data in separate packets, so that we don't send the data for a big time period all at once. Also, the used library was updated, which also decreased the memory usage.
```